test/Yesod/Auth/OAuth2Spec.hs

```diff
@@ -4,7 +4,7 @@ module Yesod.Auth.OAuth2Spec
     ) where
 
 import Test.Hspec
-import Yesod.Auth.OAuth2
+--import Yesod.Auth.OAuth2
```

Comment:
Should this be deleted entirely?

Reply:
Yeah, I can actually remove the whole file, now that there are other tests. It was here as a stub about *how* we planned to test even though we had none. It was causing `-Wall` errors to have the unused import, hence the comment. But I'll just nuke it.